detections/web/detections/web/ivanti_epm_sql_injection_remote_code_execution.yml

```diff
@@ -0,0 +1,60 @@
+data_source: []
```

Comment:
@MHaggis : can you update this yaml? the yaml keys are a bit re arranged and not consitent with others